### ISSUE TITLE
I found a bug!

### ISSUE THREAD

**Issue by aromanro** [kelseyhightower/nocode#3473]
It crashed my Windows while doing nothing.

Since I wasn't running anything else on the computer, it must be this program.

BSOD, I had to restart the computer.

**Comment by liangyiliang** [comment]
Maybe you are missing this crucial line.
` `

**Comment by aromanro** [comment]
Tried it, it still does the same. But it's worse, I think it did something irreversible to my Windows install, because I removed nocode and it still crashes.

**Comment by BertieGo** [comment]
May be not this repo problem, but u shoud examine yourself, is not using Huawei phone yet ?

**Comment by xuyifu** [comment]
Maybe you use a curried Windows？
Windows become unstable since there are more Indians in Microsoft.
PS Win7 is best

**Comment by lzzy12** [comment]
> Windows become unstable since there are more Indians in Microsoft.

Ironic enough CEO of Microsoft is an Indian Lmao

**Comment by 411906015** [comment]
why so serious~

**Comment by aromanro** [comment]
I don't know what you mean by 'serious', but it's a serious bug. If a program with no line of code can crash Windows, imagine what a two million lines program can do!

**Comment by wzj42** [comment]
You ran Windows (I bet it was Win10), that's the issue. 
Please close.

**Comment by aromanro** [comment]
That's right. But it was a portion of the windows that had no code, though.
